rust-toolchain

```diff
@@ -0,0 +1 @@
+1.45.0
```

Comment:
Hrm, I'm not sure about this one, we generally have to use nightly for our dev environments. @Nutomic ?

Reply:
Well `cargo +nightly check` is still working fine with that file present. But I'm not sure why we should specify a particular Rust version at all, as it should always work with the latest stable version (unless they break things again).

Reply:
We do see that `config-rs` is broken on **current** nightly, so it might be good to have this, but have it match our docker. Because nightly is required for the cargo formatting and clippy stuff.

Reply:
I added this file due to this error while compiling with 1.46.0: https://github.com/rust-lang/futures-rs/issues/2202

I haven't tried it again after rebasing with the isomorphic app changes. I'll try it again without this file, and remove this file if compilation succeeds. In general, if we're held back from using the latest stable version of Rust, is this method the one we'd want to go with?

Reply:
I gotcha, no its fine to leave it in, we've actually been having some problems with various rust versions too, and this would be a good way to enforce a working version for devs.